javascript/builtins/intl/NumberFormat.json

```diff
@@ -255,6 +255,120 @@
                 }
               }
             },
+            "options_roundingIncrement_parameter": {
+              "__compat": {
+                "description": "<code>options.roundingIncrement</code> parameter",
+                "support": {
+                  "chrome": {
+                    "version_added": false
+                  },
+                  "chrome_android": "mirror",
+                  "deno": {
+                    "version_added": false
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": "preview"
+                  },
+                  "firefox_android": "mirror",
```

Comment:
Is there an android nightly?

Reply:
There is: https://play.google.com/store/apps/details?id=org.mozilla.fenix&hl=en&gl=US

I use it on my phone for ages.

Reply:
Thanks. I guess we'll have to be careful with "mirror", I have another PR where stuff is added behind a preference, so "mirror" is no longer true because you can't set prefs on android.